src/db/collections/analytics/mod.rs

```diff
@@ -4,227 +4,121 @@
-            ledger_size,
-            unclaimed_tokens,
-            block_activity,
-            unlock_conditions,
-            protocol_params,
-        })
+impl<M> PerMilestone<M> {
+    fn prepare_query(&self, name: impl Into<String>) -> WriteQuery {
+        influxdb::Timestamp::from(self.milestone_timestamp)
+            .into_query(name)
+            .add_field("milestone_index", self.milestone_index)
     }
 }
 
-#[derive(Copy, Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct AddressActivityAnalytics {
-    /// The number of addresses used in the time period.
-    pub total_count: u64,
-    /// The number of addresses that received tokens in the time period.
-    pub receiving_count: u64,
-    /// The number of addresses that sent tokens in the time period.
-    pub sending_count: u64,
-}
-
-#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct AddressAnalytics {
-    pub address_with_balance_count: u64,
-}
-
-#[derive(Copy, Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct UnlockConditionAnalytics {
-    pub timelock_count: u64,
-    pub timelock_value: d128,
-    pub expiration_count: u64,
-    pub expiration_value: d128,
-    pub storage_deposit_return_count: u64,
-    pub storage_deposit_return_value: d128,
-}
-
-#[derive(Copy, Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct LedgerOutputAnalytics {
-    pub basic_count: u64,
-    pub basic_value: d128,
-    pub alias_count: u64,
-    pub alias_value: d128,
-    pub foundry_count: u64,
-    pub foundry_value: d128,
-    pub nft_count: u64,
-    pub nft_value: d128,
-    pub treasury_count: u64,
-    pub treasury_value: d128,
-}
-
-#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct LedgerSizeAnalytics {
-    pub total_storage_deposit_value: d128,
-    pub total_key_bytes: d128,
-    pub total_data_bytes: d128,
-}
-
-#[allow(missing_docs)]
-impl LedgerSizeAnalytics {
-    pub fn total_byte_cost(&self, protocol_params: &ProtocolParameters) -> d128 {
-        let rent_structure = protocol_params.rent_structure;
-        d128::from(rent_structure.v_byte_cost)
-            * ((self.total_key_bytes * d128::from(rent_structure.v_byte_factor_key as u32))
-                + (self.total_data_bytes * d128::from(rent_structure.v_byte_factor_data as u32)))
-    }
-}
-
-#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-pub struct UnclaimedTokensAnalytics {
-    pub unclaimed_count: u64,
-    pub unclaimed_value: d128,
-}
-
-#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-#[allow(missing_docs)]
-#[serde(default)]
-pub struct OutputActivityAnalytics {
-    pub alias: AliasActivityAnalytics,
-    pub nft: NftActivityAnalytics,
+/// TODO: We will need this later.
```

Comment:
I'ld either make this a `NOTE` or describe the `TODO`. I think it's better to keep `TODO`s as Gh issues rather than in the code itself.